input/fsh/EX_ExtendedExample.fsh

```diff
@@ -291,8 +289,9 @@ Instance: genomics-report-jenny-m
 InstanceOf: GenomicsReport
 Description: "Extended example: example of gene panel report"
 * status = #final "final"
-* category[0] = DiagnosticService#LAB
-* category[1] = DiagnosticService#GE
+// The category is autopopulated by SUSHI since both slices are required
+//* category[0] = DiagnosticService#LAB
+//* category[1] = DiagnosticService#GE
```

Comment:
```suggestion
// The category is autopopulated by SUSHI since the slices are required by the profile, so manually setting these, like with `* category[0] = DiagnosticService#LAB`, is not needed
```

Recommend removing commented lines of FSH in favor of just having a comment, or just omitting entirely.

Reply:
I like it as a good reminder. Maybe after a while we (or I) won't need reminding. But for now, this is my preference.